docs/DeveloperGuide.adoc

```diff
@@ -978,6 +978,24 @@ Use case resumes at step 2.
 +
 Use case resumes at step 6.
 
+[discrete]
+=== Use case: Sort a list of persons
+
+*MSS*
+
+1.  User requests to sort a list persons
```

Comment:
Correct the minor grammatical error here: `User requests to sort a list **of** persons`